### PR TITLE
feat(PS2): transactions filters

### PR DESCRIPTION
**Pull request by cwagner22**

### CODE REVIEW

File: packages/ps2/src/views/Balance/components/TransactionsHistoryTable/index.tsx [hunk 1]
```diff
@@ -29,29 +31,36 @@ const TransactionsHistoryTable = () => {
...
-        }));
-      setFilteredData(data);
+    if (transactionsEndpoint.data && coinsEndpoint.data) {
+      updateData();
     }
   }, [transactionsEndpoint.data, coinsEndpoint.data, pageIndex]);
 
+  useLayoutEffect(() => {
+    // Reset pagination when infinite query is refreshed from filter change
+    if (transactionsEndpoint.page === 1) {
```
Comment (strrife) [inline packages/ps2/src/views/Balance/components/TransactionsHistoryTable/index.tsx:59]: could we somehow put this into the infinite scroll hook?

File: packages/zignaly-ui/src/components/display/ZigTable/index.tsx [hunk 2]
```diff
@@ -181,7 +215,9 @@ export default function ZigTable<T extends object>({
               </PageNumberContainer>
               {table.getPageCount() !== -1 && (
                 <>
-                  <ZigTypography color="neutral300">out of</ZigTypography>
+                  <ZigTypography whiteSpace="nowrap" color="neutral300">
+                    out of
```
Comment (strrife) [inline packages/zignaly-ui/src/components/display/ZigTable/index.tsx:219]: could we somehow make it not use translateable strings cc @NataliaAvila-PM ?

File: packages/ps2/src/apis/coin/types.ts [hunk 3]
```diff
@@ -58,29 +58,27 @@ export type DepositInfo = {
   };
 };
 
-export enum TransactionType {
+export const TRANSACTION_TYPE = {
```
[the remark this answers is not in the document]
Comment (cwagner22) [reply packages/ps2/src/apis/coin/types.ts:61]: I prefer to work with objects in this case: https://youtu.be/jjMbPt_H3RQ

File: packages/ps2/src/views/Balance/components/TransactionsHistoryTable/index.tsx (hunk 1 above)
Comment (cwagner22) [reply packages/ps2/src/views/Balance/components/TransactionsHistoryTable/index.tsx:59]: The problem is we need to reset the table pagination when filter changes so I think this way is needed

**Pull request merged**